### PR TITLE
[tools/onert_train] Revisit args in onert_train

### PR DESCRIPTION
**Pull request by YongseopKim**
This commit revisits args code to fix typo and remove unnecessary new lines.

ONE-DCO-1.0-Signed-off-by: Yongseop Kim <yons.kim@samsung.com>

### CODE REVIEW

**Comment by YongseopKim** [comment]
Now

```diff
1,4c1
< onert_train
<
< Usage: ./Product/out/bin/onert_train[model path] [<options>]
<
---
> Usage: ./Product/out/bin/onert_train [model path] [<options>]
18d14
<
21,22c17
<
<   -m [ --mem_poll ] arg (=0)            Check memory polling
---
>   -m [ --mem_poll ] arg (=0)            Check memory polling (default: false)
29d23
<
34d27
<
38,40c31,32
<
<   --metric arg (=-1)                    Metricy type
<                                           Simply calculates the metric value using the variables
---
>   --metric arg (=-1)                    Metric type
>                                         Simply calculates the metric value using the variables
43,45c35,36
<
<   --validation_split arg (=0)           Float between 0 and 1. Fraction of the training data to be
<                                         used as validation data.
---
>   --validation_split arg (=0)           Float between 0 and 1(0 < float < 1). Fraction of the
>                                         training data to be used as validation data.
50d40
<
```

**Comment by YongseopKim** [comment]
PTAL @jyoungyun and @Samsung/one_onert

File: tests/tools/onert_train/src/args.cc [hunk 1]
```diff
@@ -267,8 +267,7 @@ void Args::Parse(const int argc, char **argv)
 
   if (vm.count("help"))
   {
-    std::cout << "onert_train\n\n";
-    std::cout << "Usage: " << argv[0] << "[model path] [<options>]\n\n";
+    std::cout << "Usage: " << argv[0] << " [model path] [<options>]\n";
```
Comment (jyoungyun) [inline tests/tools/onert_train/src/args.cc:270]: I would like to maintain the same usage format as `onert_run` to have unity for ONE users. Could you keep this code as it is or modify `onert_run` like this? :)

Comment (YongseopKim) [reply tests/tools/onert_train/src/args.cc:270]: Which format you want to keep?
0. `std::cout << "Usage: " << argv[0] << "[model path] [<options>]\n\n";` (original)
1. `std::cout << "Usage: " << argv[0] << " [model path] [<options>]\n\n";` (add space)
2. `std::cout << "Usage: " << argv[0] << "[model path] [<options>]\n";` (remove one new line)

Comment (YongseopKim) [reply tests/tools/onert_train/src/args.cc:270]: FYI,

from now on wo this PR,
```
$ ./Product/out/bin/onert_train --help
onert_train

Usage: ./Product/out/bin/onert_train[model path] [<options>]
```

**Review by zetwhite — DISMISSED** [review]
LGTM :+1:

File: tests/tools/onert_train/src/args.cc (hunk 1 above)
Comment (jyoungyun) [reply tests/tools/onert_train/src/args.cc:270]:
```
$ ./Product/out/bin/onert_run --help
onert_run

Usage: ./Product/out/bin/onert_run path to nnpackage root directory [<options>]


General options:
```

The current code is the same with `onert_run`.

Comment (YongseopKim) [reply tests/tools/onert_train/src/args.cc:270]: I got it!

**Comment by YongseopKim** [comment]
FYI, this PR might conflict with https://github.com/Samsung/ONE/pull/12419

**Comment by YongseopKim** [comment]
PTAL

**Review by zetwhite — APPROVED** [review]

**Review by jyoungyun — APPROVED** [review]
LGTM Thank you!

**Pull request merged**